modules/codemod/lib/v6/deprecateCookieBanner.ts

```diff
@@ -1,124 +1,15 @@
-  root
-    .find(j.ImportDeclaration, {source: {value: '@workday/canvas-kit-react/cookie-banner'}})
-    .forEach(nodePath => {
-      nodePath.value.specifiers?.forEach(specifier => {
-        if (specifier.type === 'ImportDefaultSpecifier') {
-          //  `import CookieBanner` becomes `import DeprecatedCookieBanner`
-          if (specifier.local?.name === 'CookieBanner') {
-            specifier.local.name = 'DeprecatedCookieBanner';
-          }
-        }
-        if (specifier.type === 'ImportSpecifier') {
-          //  `import {CookieBanner}` becomes `import {DeprecatedCookieBanner}`
-          if (specifier.imported.name === 'CookieBanner') {
-            specifier.imported.name = 'DeprecatedCookieBanner';
-          }
-          //  `import {CookieBannerProps}` becomes `import {DeprecatedCookieBannerProps}`
-          if (specifier.imported.name === 'CookieBannerProps') {
-            specifier.imported.name = 'DeprecatedCookieBannerProps';
-          }
-        }
-        return specifier;
-      });
-    });
-
-  // Transform CookieBannerProps type references
-  // e.g. `type CustomProps = CookieBannerProps;` becomes `type CustomProps = DeprecatedCookieBannerProps;`
-  root
-    .find(j.TSTypeReference, {typeName: {type: 'Identifier', name: 'CookieBannerProps'}})
-    .forEach(nodePath => {
-      const identifier = nodePath.value.typeName as Identifier;
-      identifier.name = 'DeprecatedCookieBannerProps';
-    });
-
-  // Transform CookieBannerProps type interface declaration references
-  // e.g. `interface CustomProps extends CookieBannerProps` becomes `interface CustomProps extends DeprecatedCookieBannerProps`
-  root.find(j.TSInterfaceDeclaration).forEach(nodePath => {
-    // If the interface is extending CookieBannerProps, transform the extension name to DeprecatedCookieBannerProps
-    nodePath.node.extends?.forEach(typeExtension => {
-      if (
-        typeExtension.expression.type === 'Identifier' &&
-        typeExtension.expression.name === 'CookieBannerProps'
-      ) {
-        typeExtension.expression.name = 'DeprecatedCookieBannerProps';
-      }
-    });
-  });
-
-  // Transform CookieBanner JSXElements
-  // e.g. `<CookieBanner>` becomes `<DeprecatedCookieBanner>`
-  root.find(j.JSXIdentifier, {name: 'CookieBanner'}).forEach(nodePath => {
-    nodePath.node.name = 'DeprecatedCookieBanner';
-  });
 
-  // Transform CookieBanner MemberExpressions
-  // e.g. `CookieBanner.DefaultNotice` becomes `DeprecatedCookieBanner.DefaultNotice`
-  root
-    .find(j.MemberExpression, {object: {type: 'Identifier', name: 'CookieBanner'}})
-    .forEach(nodePath => {
-      const identifier = nodePath.value.object as Identifier;
-      identifier.name = 'DeprecatedCookieBanner';
-    });
+  return renameImports(api, root, '@workday/canvas-kit-react/cookie-banner', {
+    CookieBanner: 'DeprecatedCookieBanner',
+    CookieBannerProps: 'DeprecatedCookieBannerProps',
+  }).toSource();
+};
```

Comment:
I ran into type errors in v6 codemod files because of the upgrade of `jscodeshift` from `0.11.0` to `0.13.1`. The Typescript types were tightened and all these `find` commands didn't return a `boolean`. As I was looking to fix them all, I noticed almost all the codemods renamed import specifiers. So I made a utility to do this.